lib/sys/monitor.js

```diff
@@ -17,7 +16,7 @@ function secretInvalid(req, res) {
   return false
 }
 
-function setRoutes(server) {
+function setRoutes({ rateLimit }, server) {
```

Comment:
Inject config, eliminating one of the few places `server-config` is still imported outside of the server start script.